sklearn/feature_selection/_univariate_selection.py

```diff
@@ -640,7 +640,7 @@ class SelectFpr(_BaseFilter):
         works with classification tasks.
 
     alpha : float, default=5e-2
-        The highest p-value for features to be kept.
+        The lowest p-value for features to not be kept.
```

Comment:
I still think the adjustment a little unclear. What do you think of the following?

```suggestion
        Features with p-values less than `alpha` are selected.
```